docs/blockchain-integrations-framework/index.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: nodes.liquid
+section: bif
+date: Last Modified
+title: "Blockchain Integrations Framework"
+permalink: "docs/blockchain-integrations-framework/"
+---
+
+Chainlink Labs provides self-serve tools that node operators and blockchain projects can use to deploy Data Feed integrations on EVM networks. These tools are called the Blockchain Integrations Framework (BIF). This framework includes a command line interface (CLI) and is available as a binary file. The binary is not publicly available. During the Alpha release, you must receive this binary directly from Chainlink Labs.
+
+The framework simplifies the following tasks:
+
+- Deploy Chainlink nodes into Kubernetes clusters and distribute jobs to them
+- Run tests that generate reports detailing compatibility between Chainlink Labs' EVM integration and a given blockchain
+- Deploy cluster infrastructure required to test OCR for Data Feeds
+- Perform longer running soak tests for OCR for Data Feeds
+- Perform smart contract management tasks with the same capabilities as Chainlink Labs' tools
+
+**Contents**
+
+- [Requirements](#requirements)
+- [Set up your environment](#set-up-your-environment)
+- [Running basic tests](#running-basic-tests)
+- [Running integration testing](#running-integration-tests)
+- [Troubleshooting integration tests](#troubleshooting-integration-tests)
+
+## Requirements
+
+Most basic tests can be completed on a system with minimal hardware resources. To complete soak testing and use the CLI to create clusters with multiple Chainlink nodes, you must run a Kubernets cluster on a system with the following system resources:
```

Comment:
I think we should mention the Docker requirement here, and say that Docker must be allotted at least 6 CPU and 10 GB RAM.

Reply:
I added Docker here and also in the environment setup instructions with a link to https://docs.docker.com/engine/install/

How do you configure Docker default limits? I'm not seeing it in the docs, and it looks like Docker uses all available CPUs and memory by default: https://docs.docker.com/config/containers/resource_constraints/#cpu

Reply:
By default, any Docker container is given unlimited access to the resources available to Docker. But Docker may only be allotted 4CPUs and 8GB RAM, which would be insufficient for our purposes. I haven't found a way to boost these limits without Docker Desktop. Here are the Linux instructions: https://docs.docker.com/desktop/settings/linux/

Reply:
Added in the most recent commit. I added info in both the Resources section and the environment setup instructions.